resource/storage.go

```diff
@@ -20,7 +20,7 @@ type Storer interface {
 	// If the fetching of the data is not immediate, the method must listen for cancellation
 	// on the passed ctx. If the operation is stopped due to context cancellation, the
 	// function must return the result of the ctx.Err() method.
-	Find(ctx context.Context, lookup *Lookup, page, perPage int) (*ItemList, error)
+	Find(ctx context.Context, lookup *Lookup, page, perPage int, skip int) (*ItemList, error)
```

Comment:
This is a breaking change. I'm not against it but we need to at least update all the supported storage handlers:

* https://github.com/rs/rest-layer-mem
* https://github.com/rs/rest-layer-mongo
* https://github.com/rs/rest-layer-es
* https://github.com/rs/rest-layer-hystrix

Alternatively we may think of an optional interface to introduce this change. But on the long run it would add complexity so a breaking but supported change is fine by me.